test/entt/entity/observer.cpp

```diff
@@ -369,3 +369,34 @@ TEST(Observer, GroupCornerCase) {
     ASSERT_FALSE(add_observer.empty());
     ASSERT_TRUE(remove_observer.empty());
 }
+
+enum class CustomEntity : std::uint32_t
```

Comment:
I know you may not like the coding style but EnTT uses snake case for types everywhere. Please be consistent. 🙂